js/adapt-contrib-slider.js

```diff
@@ -370,15 +370,24 @@ define([
             var $markers = this.$('.slider-markers').empty();
             if (this.model.get('_showScale') === false) {
                 $markers.eq(0).css({display: 'none'});
-                this.$('.slider-scale-numbers').eq(0).css(
+                this.$('.slider-scale-numbers *:not(".slider-scale-marker")').css(
                     this.model.get('_showScaleIndicator') ? {visibility: 'hidden'} : {display: 'none'}
                 );
             } else {
                 var $scaler = this.$('.slider-scaler');
-                for (var i = 0, count = this.model.get('_items').length; i < count; i++) {
-                    $markers.append("<div class='slider-line component-item-color'>");
-                    $markers.find('.slider-line').eq(i).css({left: this.mapIndexToPixels(i, $scaler) + 'px'});
+                for (var i = 1, count = this.model.get('_items').length - 1; i < count; i++) {
+                    $markers.append("<div class='slider-line component-item-color' style='left: " + this.mapIndexToPixels(i, $scaler) + "px'>");
                 }
+                // Do we show scale numbers
+                this.showScaleNumbers();
+            }
+        },
+
+        showScaleNumbers: function () {
+            var $scaler = this.$('.slider-scaler');
+            if (this.model.get('_showScaleNumbers') === false) {
+                this.$('.slider-scale-numbers *:not(".slider-scale-marker")').css('display', 'none');
+            } else {
```

Comment:
again you could `return` here and avoid having to nest the rest of the code in an `else` statement